heron/config/src/yaml/BUILD

```diff
@@ -14,7 +14,7 @@ filegroup(
 
 filegroup(
     name = "conf-yaml",
-    srcs = glob(["conf/**/*.yaml"]),
+    srcs = glob(["conf/**/*.yaml"] + ["conf/**/*.aurora"]),
```

Comment:
there is `conf-aurora-yaml` why not use that

Reply:
@huijunw - this target ```conf-yaml``` includes every yaml file and also every aurora file.